__tests__/unit/components/app/TourForm.test.tsx

```diff
@@ -0,0 +1,76 @@
+import '@testing-library/jest-dom'
+import { ConnectionStatusProvider } from '../../../../src/components/providers/ConnectionStatusProvider'
+import React from 'react'
+import 'reflect-metadata'
+import TourForm from '../../../../src/components/app/TourForm'
+import { Point } from 'geojson'
+import { Tour } from '../../../../src/types/tour'
+import { render } from '@testing-library/react'
+import { MemoryRouter } from 'react-router'
+
+const mockConnectionStatusContext: any = {
+  isOffline: () => false
+}
+const mockImageUploadContext: any = {
+  files: [],
+  currentUploads: []
+}
```

Comment:
Why not typehint the object properly? You could still only specify those props you need by using `as XyzContext`.

Reply:
How would you do that so I can only specify the props I need? Adding the specific type will always ask for all of the properties to be defined unless you use some hacky solution. Or at least that's what I experienced so far. Let's talk about this in one of our meetings. :)